dm/dm/config/task.go

```diff
@@ -1103,6 +1110,7 @@ func NewSyncerConfigsForDowngrade(syncerConfigs map[string]*SyncerConfig) map[st
 			EnableGTID:              syncerConfig.EnableGTID,
 			DisableCausality:        syncerConfig.DisableCausality,
 			SafeMode:                syncerConfig.SafeMode,
+			SafeModeResumeDuration:  syncerConfig.SafeModeResumeDuration,
```

Comment:
This function is `NewSyncerConfigsForDowngrade` ! lower versions don't have `safe-mode-resume-duration`, so we should not write it.